src/Modules/OrchardCore.Commerce/Abstractions/IStripePaymentService.cs

```diff
@@ -31,9 +31,15 @@ public interface IStripePaymentService
     Task<ContentItem> CreateOrUpdateOrderFromShoppingCartAsync(PaymentIntent paymentIntent, IUpdateModelAccessor updateModelAccessor);
 
     /// <summary>
-    /// Updates the corresponding order status to Order for the given <paramref name="paymentIntent"/>.
+    /// Creates an order content item without payment in the database based on the current <see cref="ShoppingCart"/> content.
     /// </summary>
-    Task UpdateOrderToOrderedAsync(PaymentIntent paymentIntent);
+    Task<ContentItem> CreateNoPaymentOrderFromShoppingCartAsync(IUpdateModelAccessor updateModelAccessor);
```

Comment:
It's not clear why is this in the Stripe payment service? It's not Stripe-specific. I think it should be in the `IPaymentService` so it won't break when we incorporate the option for other payment providers.
(This method is different from `UpdateOrderToOrderedAsync` that actually Stripe stuff, which is fine for now.)

Reply:
While at it, I also moved `CreateOrderLineItemsAsync()` to `IPaymentService` as it seems better to have `StripePaymentService` depend on `PaymentService` than vice-versa.